lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -630,11 +678,16 @@ void SessionState::get_updates(
   get_event_trigger_updates(update_request_out, actions_out, update_criteria);
 }
 
+void SessionState::mark_as_awaiting_termination(
```

Comment:
I think I deleted this function, is this a bad rebase?

Reply:
... may be ..  once the  2 dependeant PR (ronit amf proto + sanjay 3025)  merged to master, I wud be clear about my exact code changes, I will resolve in my next commit

Reply:
removed